memory_leak_worker_logging/evm_mem_log_processor.rb

```diff
@@ -78,18 +84,17 @@
 
   io_klass.open(log_file) do |file|
     file.each_line do |line|
-      next unless LOG_LINE_REGEXP =~ line
-      next if options[:pid] && options[:pid] != Regexp.last_match[5]
+      next unless (m=line.match(LOG_LINE_REGEXP))
```

Comment:
Can we change this to `line_match`?  I don't really like single letter variable names, and there is no reason we can't have this a longer name.

Pedantic (though, that is pretty much this entire PR):  can we also put spaces around the `=` and remove the extra parens around the whole thing?  Unlike our defined rubocop rules in MIQ, I do like avoiding parens whenever possible in my own ruby code.

Reply:
done